### ISSUE TITLE
Duplicate set element without duplicate

### ISSUE THREAD

**Issue by nikolavojicic** [clj-kondo/clj-kondo#1173]
clj-kondo-2021.02.13-windows-amd64
Java 11, Emacs + flycheck-clj-kondo

These incorrectly report `duplicate set element " "`:
#{"–" "•"}
#{"—" "•"}
#{"—" "–"}

**Comment by borkdude** [comment]
@nikolavojicic I cannot reproduce this when pasting your code into a `foo.clj` file and then running:

```
$ clj-kondo --lint /tmp/foo.clj
linting took 8ms, errors: 0, warnings: 0
```

Can you maybe upload this code in a file, in order to prevent encoding issues?

**Comment by nikolavojicic** [comment]
You are right, this was an encoding issue.
Should have tried it from console...
It works fine.
